changelog/fix_make_rails_time_zone_aware_of_timezone_offset.md

```diff
@@ -0,0 +1 @@
+* [#800](https://github.com/rubocop/rubocop-rails/issues/800): This PR makes `Rails/TimeZone` aware of timezone UTF offset. ([@inkstak][])
```

Comment:
```suggestion
* [#800](https://github.com/rubocop/rubocop-rails/issues/800): Make `Rails/TimeZone` aware of timezone UTF offset. ([@inkstak][])
```

Reply:
Can you rename file from `changelog/fix_make_rails_time_zone_aware_of_timezone_offset.md` to `changelog/change_make_rails_time_zone_aware_of_timezone_offset.md`?